virttest/qemu_vm.py

```diff
@@ -611,7 +611,8 @@ def add_smp(devices):
         def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     netdev_id=None, nic_extra_params=None, pci_addr=None,
                     bootindex=None, queues=1, vectors=None, pci_bus='pci.0',
-                    ctrl_mac_addr=None):
+                    ctrl_mac_addr=None, indirect_desc=None, mrg_rxbuf=None,
+                    ctrl_vq=None, ctrl_vlan=None, status=None):
```

Comment:
Recommend use `**kwargs` instead, this way we don't need further modification when there is other new option.